source/include/ota.h

```diff
@@ -323,6 +323,11 @@ typedef struct OtaAgentContext
 #define OTA_ERR_SUBSCRIBE_FAILED             0x40000000U /*!< Failed to subscribe to a topic. */
 #define OTA_ERR_UNSUBSCRIBE_FAILED           0x41000000U /*!< Failed to unsubscribe from a topic. */
 #define OTA_ERR_FAILED_TO_DECODE_CBOR        0x42000000U /*!< Failed to decode CBOR object. */
+#define OTA_ERR_TIMER_CREATE_FAILED          0x43000000U /*!< Failed to create timer. */
```

Comment:
Why not use the same error codes as
https://github.com/aws/ota-for-aws-iot-embedded-sdk/blob/38f3af7d4e6219961fcacc0c2bc56eaea47252b8/source/include/ota.h#L319 ?

Reply:
Yes seems a mistake

Reply:
Addressed in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/91/commits/935a2ddcdf079e6aca118c6dd4266df35a825df0